src/DbSchemaManager.php

```diff
@@ -5,59 +5,81 @@
 namespace Yiisoft\Cache\Db;
 
 use Throwable;
+use Yiisoft\Db\Command\CommandInterface;
 use Yiisoft\Db\Connection\ConnectionInterface;
 use Yiisoft\Db\Exception\Exception;
 use Yiisoft\Db\Exception\InvalidConfigException;
 use Yiisoft\Db\Exception\NotSupportedException;
 use Yiisoft\Db\Schema\SchemaInterface;
 
+/**
+ * Manages the cache table schema in the database.
+ */
 final class DbSchemaManager
 {
+    private CommandInterface $command;
+    private SchemaInterface $schema;
+
     public function __construct(private ConnectionInterface $db)
     {
+        $this->command = $db->createCommand();
+        $this->schema = $db->getSchema();
```

Comment:
I don't think it is a good idea to do anything like this in constructor:

1. Initialization should be postponed. In this case it doesn't really matter but if it is performance-heavy it is a must.
2. `getSchema()` is very light so caching its instance doesn't make much sense.